data/successors/successor ships.txt

```diff
@@ -1121,6 +1130,8 @@ ship "Vujlet"
 	gun 6 -46 `"Ojet" Bimodal Coilgun`
 	gun -9 -30
 	gun 9 -30
+	pylon -9 26
+	pylon 9 26
```

Comment:
Same as the Seiitej, I think it would make the most sense for there to be a single pylon on the ship's midline instead.